dbt/version.py

```diff
@@ -1,43 +1,31 @@
+import json
 import re
 
-import dbt.semver
-
-try:
-    # For Python 3.0 and later
-    from urllib.request import urlopen
-except ImportError:
-    # Fall back to Python 2's urllib2
-    from urllib2 import urlopen
+import requests
 
-REMOTE_VERSION_FILE = \
-    'https://raw.githubusercontent.com/fishtown-analytics/dbt/' \
-    'master/.bumpversion.cfg'
+import dbt.exceptions
+import dbt.semver
 
 
-def get_version_string_from_text(contents):
-    matches = re.search(r"current_version = ([\.0-9a-z]+)", contents)
-    if matches is None or len(matches.groups()) != 1:
-        return ""
-    version = matches.groups()[0]
-    return version
+PYPI_VERSION_URL = 'https://pypi.org/pypi/dbt/json'
 
 
-def get_remote_version_file_contents(url=REMOTE_VERSION_FILE):
+def get_latest_version():
+    resp = requests.get(PYPI_VERSION_URL)
```

Comment:
Can we make this just report "<unknown>" if the pypi request fails? That could happen with:
1. pypi downtime
2. wifi turned off (weird, but it... has happened)
3. restrictive firewall rules

The stacktrace you get right now if this request fails is frightening. I'd prefer to just show the local version if a user is in one of these failure modes. We could log the full error at a debug level.